.travis.yml

```diff
@@ -10,11 +10,13 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u github.com/stretchr/testify
+  - go get -u github.com/GoASTScanner/gas
```

Comment:
@marstr It seems that the github.com/HewlettPackard/gas repository was deleted from github. Probably you should update the azure-sdk-for-go to use GoASTScanner.